addon-test-support/@ember/test-helpers/dom/trigger-event.ts

```diff
@@ -72,13 +72,17 @@ export default function triggerEvent(
         throw new Error(`Must provide an \`eventType\` to \`triggerEvent\``);
       }
 
-      let element = getElement(target);
+      let element = getTarget(target);
       if (!element) {
         throw new Error(`Element not found when calling \`triggerEvent('${target}', ...)\`.`);
       }
 
-      if (isFormControl(element) && element.disabled) {
-        throw new Error(`Can not \`triggerEvent\` on disabled ${element}`);
+      if (!isWindow(element)) {
```

Comment:
Let's drop the helper function, and do the check here directly (I don't think the helper function gives us anything useful).

```js
let element = target instanceof Window ? target : getElement(target);
if (!element) {
  throw new Error(`Element not found when calling \`triggerEvent('${target}', ...)\`.`)
}
```

Reply:
It's useful for using it in a few different helpers, such as `click` and `doubleClick`. I have to add the same check there, so this helper function was in preparation for that. The next commit should make it more clear (if the TODO didn't).

Reply:
See recent updates.